easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.3.0-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
```

Comment:
@boegel ah, that explains a few things :) But in the other easyconfig files no mention is made to that easyblock, is it automatically chosen from the easyconfig name?